.circleci/config.yml

```diff
@@ -30,23 +30,14 @@ workflows:
   simpleprophet-build:
     jobs:
     - gcp-gcr/build-and-push-image:
-        # This step relies on several environment variables provided to the
-        # build job via CircleCI configuration:
-        #   GCLOUD_SERVICE_KEY
-        #   GOOGLE_PROJECT_ID
-        #   GOOGLE_COMPUTE_ZONE
-        # We provisioned a dataops sandbox project specifically for this
-        # repository, enabled the GCR API, and provisioned a service account
-        # with read and write access to GCS per
-        #   https://cloud.google.com/container-registry/docs/access-control
-        # We also have marked the repository in GCR as public via the GCP console
-        # so that arbitrary users and applications can access the images.
+        # Context defined in https://bugzilla.mozilla.org/show_bug.cgi?id=1608958
+        context: data-eng-airflow-gcr
```

Comment:
I will also delete the environment variables associated with this project in CircleCI before merging to avoid confusion, since the context will override them.